theseus_gui/src/pages/Settings.vue

```diff
@@ -1,7 +1,319 @@
+              Test
+            </Button>
+          </span>
+        </div>
+      </div>
+      <div class="settings-group">
+        <h3>Java 8 Location</h3>
+        <div class="toggle-setting">
+          <input
+            v-model="settings.java_globals.JAVA_8.path"
+            type="text"
+            class="input installation-input"
+            placeholder="/path/to/java8"
+          />
+          <span class="installation-buttons">
+            <Button @click="saveJavaPath">
+              <SearchIcon />
+              Auto Detect
+            </Button>
+            <Button @click="saveJavaPath">
+              <BrowseIcon />
+              Browse
+            </Button>
+            <Button @click="saveJavaPath">
+              <PlayIcon />
+              Test
+            </Button>
+          </span>
+        </div>
+      </div>
+      <hr class="card-divider" />
+      <div class="settings-group">
+        <h3>Java Arguments</h3>
+        <input
+          v-model="settings.custom_java_args"
+          type="text"
+          class="input installation-input"
+          placeholder="-Xms10G -Xmx10G -XX:+UseG1GC -XX:+ParallelRefProcEnabled"
+        />
+      </div>
+      <div class="settings-group">
+        <h3>Environment Arguments</h3>
+        <input
+          v-model="settings.custom_env_args"
+          type="text"
+          class="input installation-input"
+          placeholder="-Xms10G -Xmx10G -XX:+UseG1GC -XX:+ParallelRefProcEnabled"
+        />
+      </div>
+      <hr class="card-divider" />
+      <div class="settings-group">
+        <div class="sliders">
+          <span class="slider">
+            Minimum Memory
+            <Slider v-model="settings.memory.minimum" :min="1024" :max="8192" :step="1024" />
+          </span>
+          <span class="slider">
+            Maximum Memory
+            <Slider v-model="settings.memory.maximum" :min="1024" :max="8192" :step="1024" />
+          </span>
+        </div>
+      </div>
+    </Card>
+    <Card class="settings-card">
+      <h2 class="settings-title">Window Size</h2>
+      <div class="settings-group">
+        <div class="settings-group">
+          <div class="sliders">
+            <span class="slider">
+              Width
+              <Slider v-model="settings.game_resolution[0]" :min="400" :max="2562" :step="2" />
+            </span>
+            <span class="slider">
+              Height
+              <Slider v-model="settings.game_resolution[1]" :min="400" :max="2562" :step="2" />
+            </span>
+          </div>
+        </div>
+        <hr class="card-divider" />
+        <div class="settings-group">
+          <h3>Console</h3>
+          <div class="toggle-setting">
+            Maximum Concurrent Downloads
+            <input
+              v-model="settings.max_concurrent_downloads"
+              type="text"
+              name="concurrent-downloads"
+              class="concurrent_downloads"
+            />
+          </div>
```

Comment:
this seems like a pretty weird section to put launcher-specific settings in. IMO it should just be it's own card called "Launcher settings"